README.md

```diff
@@ -248,21 +249,21 @@ There are several ways to get help with installing and using gEDA:
   <http://wiki.geda-project.org/geda:mailinglists>
 
 - If you have discovered a bug, have a feature request, or have
-  written a patch to lepton-eda, please create an item on the
-  lepton-eda bug tracker page:
+  written a patch to Lepton, please create an item on the lepton-eda
+  bug tracker page:
   <https://github.com/lepton-eda/lepton-eda/issues>
 
 License
 =======
 
-gEDA/gaf (this package) is freely distributable under the GNU Public
+Lepton EDA (this package) is freely distributable under the GNU Public
 License (GPL) version 2.0 or (at your option) any later version.  See
 the `COPYING` file for the full text of the license.
 
 The programs and associated files are:
 
-  Copyright (C) 1998-2013 by Ales Hvezda and the respective original
+  Copyright (C) 1998-2017 by Ales Hvezda and the respective original
```

Comment:
I think here you could add "Copyright (C) 2017 Lepton Developers" (or contributors) to prevent changing the existing line in future.